test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -797,6 +797,32 @@ public void WorkspaceContextMenu_IfSubmenuOpenOnMouseHover()
             Assert.IsTrue(currentWs.WorkspaceLacingMenu.IsSubmenuOpen);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestDumpLibraryToXml()
+        {
+            ViewModel.DumpLibraryToXmlCommand.Execute(null);
+
+            // Look for library dump files aged within 10 seconds from the current time
+            var time = System.DateTime.Now;
+            var timeout = 10;
+
+            for (var t = 0; t < timeout; t++)
```

Comment:
I'd write this as `for (var attempt = 0; attempt < attempts; ++attempt)`, just a small suggestion.